ropetest/codeanalyzetest.py

```diff
@@ -432,7 +432,7 @@ def test_renaming_functions_with_from_import_and_parens(self):
         found_pyname = name_finder.get_pyname_at(code.index('afunc') + 1)
         self.assertEquals(afunc.get_object(), found_pyname.get_object())
 
-    @testutils.run_only_for_25
```

Comment:
> is 2.5 supported? If not, perhaps these tests should be removed? :s

From time to time (not very often, I am afraid) I pull out a virtual machine with RHEL-5 (that's python 2.4) and try to run the tests. Last time I tried (and that is some time, true) all tests passed. But perhaps, it is really the time. @aligrudi , what do you think, should we kill python < 2.6?